File: Python Edition/cybcalc.py
import time

def loading():
	animation = ['|', '/', '-', '\\']
	for i in range (6):
		print ("CYBERCALCULATOR STARTING: ", i * 20, "%", animation[i % 4], end='')
		print ('\r', end='')
		time.sleep(1)
	print (" ")

print (" ")
print ("<SYSYEM REBOOTING>")
time.sleep(2)
print ("[###             ]")
time.sleep(1)
print ("[#######         ]")
time.sleep(1)
print ("[################]")
time.sleep(1)
print ("<REBOOTING SUCCESSFUL>")
time.sleep(1)
print (" ")
print ("Johnny: Wake the fuck up samurai. We've got a city to burn.")
print (" ")
time.sleep(4)

loading()

time.sleep(1)
print (" ")
print (" ####   ####  ##      ####  ##  ## ##      ####  ######  ####  #####")
print ("##  ## ##  ## ##     ##  ## ##  ## ##     ##  ##   ##   ##  ## ##  ##")
print ("##     ###### ##     ##     ##  ## ##     ######   ##   ##  ## #####")
print ("##  ## ##  ## ##     ##  ## ##  ## ##     ##  ##   ##   ##  ## ##  ##")
print (" ####  ##  ## ######  ####   ####  ###### ##  ##   ##    ####  ##  ##")
print (" ")

act = input ("Select act: ")
a = float(input ("Enter first number: "))
b = float(input ("Enter second number: "))
if act == "+":
	c = a + b
elif act == "-":
	c = a - b
elif act == "*":
	c = a * b
elif act == "/":
	c = a / b
elif act == "%":
	c = a % b
elif act == "**":
	c = a ** b
elif act == "//":
	c = a // b
print (" ")

print ("Result: " + str(c))


# Created by h4cktivist
# In support of CYBERPUNK 2077

File: C# Edition/cybcalc.cs
using System;

public class CyberCalculator {
	public static void Main(String[] args) {

		Console.WriteLine(" ");
		Console.WriteLine("Johnny: Wake the fuck up samurai. We've got a city to burn.");
		Console.WriteLine(" ");
		Console.WriteLine(" ####   ####  ##      ####  ##  ## ##      ####  ######  ####  #####");
		Console.WriteLine("##  ## ##  ## ##     ##  ## ##  ## ##     ##  ##   ##   ##  ## ##  ##");
		Console.WriteLine("##     ###### ##     ##     ##  ## ##     ######   ##   ##  ## #####");
		Console.WriteLine("##  ## ##  ## ##     ##  ## ##  ## ##     ##  ##   ##   ##  ## ##  ##");
		Console.WriteLine(" ####  ##  ## ######  ####   ####  ###### ##  ##   ##    ####  ##  ##");
		Console.WriteLine(" ");

		Console.Write("What do you want to do? (+,-,/,*, %): ");
		string act = Console.ReadLine();
		Console.Write("Enter first number: ");
		double num1 = Convert.ToDouble(Console.ReadLine());
		Console.Write("Enter second number: ");
		double num2 = Convert.ToDouble(Console.ReadLine());
		Console.WriteLine(" ");
		
		switch(act)
		{
			case "+":
				Console.WriteLine($"Result is: {num1 + num2}");
				break;
			case "-":
				Console.WriteLine($"Result is: {num1 - num2}");
				break;
			case "/":
				Console.WriteLine($"Result is: {num1 / num2}");
				break;
			case "*":
				Console.WriteLine($"Result is: {num1 * num2}");
				break;
			case "%":
				Console.WriteLine($"Result is: {num1 % num2}");
				break;
			default:
				Console.WriteLine("Wrong action selected!");
				break;
		}

		Console.WriteLine(" ");
		Console.WriteLine("Thanks for using it! (c)h4cktivist");
	}
}


// created by h4cktivist